Formula/tokei.rb

```diff
@@ -13,7 +13,7 @@ class Tokei < Formula
   depends_on "rust" => :build
 
   def install
-    system "cargo", "install", "--root", prefix
+    system "cargo", "install", "--root", prefix, "--feaures", "all"
```

Comment:
Looks like there's a typo here, in `feaures`.

Reply:
Fixed in 27e5a9b.